### DISCUSSION TITLE
Support of web workers

### DISCUSSION THREAD

**Discussion by t1m0t** [framesurge/perseus#127] [General]
Hi,
First thanks for this framework. Very good work done.
As introduce in the title, does Perseus handle web workers? Also, can `gloo_worker` be used with Perseus?

**Comment by arctic-hen7** [comment]
Thank you very much! As yet, Perseus doesn't use web workers internally, and I haven't tried using them with it yet (though PWAs are on the roadmap, so that'll happen at some point).

That said, I don't see any reason why they wouldn't work, as you can host arbitrary static glue code files, JS snippets are supported automatically, and everything's built on `wasm-bindgen`. If you have any problems, I'm happy to help, and I'd love to see this as an example in the repo if you manage it!
